README.md

```diff
@@ -1,2 +1,12 @@
-# lyricfeel
-Visualiza las emociones de las letras de tus canciones favoritas y conoce cómo se sienten tus amigos.
+# Bibliofetch
+
+Ricardo Ruiz Fernández de Alba
+
+## Descripción del problema
+
+Dadas una serie de asignaturas de la Universidad de Granada puede ser tedioso o inaccesible conocer y acceder al material bibliográfico requerido, puesto que en muchos casos, el apartado de la guía docente no se encuentra en un formato estucturado o puede contener enlaces caídos.
```

Comment:
Te rogaría que no pensaras que me acuerdo de lo que hemos estado hablando. En este caso es así, pero se trata de que tratéis de crear un documento lo más concreto posible, si puede ser basado en el juego de rol, donde entre otras cosas hay que identificar el cliente. ¿Estamos hablando de un estudiante de Letras? (donde hay 60 libros en la biblio, a veces repetido, y algunos obligatorios se agotan en biblioteca rápido? ¿O uno de informática? ¿Puedes enlazar una guía docente de ejemplo?

Cuanta más información metas ahora, menos tendrás que inventarte en el objetivo siguiente.

Reply:
En la identificación al cliente como estudiante de Letras, me encuentro con el problema del requisito del conocimiento personal. Quizás tenga más sentido orientarlo a un estudiante genérico de la Universidad de Granada y detallar en el futuro próximo en el Objetivo 1 los tipos de usuario ("personas") y sus Historias de usuario incluyendo entre ellas las necesidades específicas de las humanidades.